infra/terraform/container-definitions-thumbor.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "name": "thumbor",
+    "image": "wellcome/wellcomecollection-thumbor:test",
+    "cpu": 512,
+    "memory": 1000,
```

Comment:
I’d have to check, but isn’t the max memory on an instance a multiple of 1024? ECS surely won’t be able schedule a second container in that remaining 24 MiB, so it’s just wasted.